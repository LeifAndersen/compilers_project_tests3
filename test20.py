def b():
 a = 3

