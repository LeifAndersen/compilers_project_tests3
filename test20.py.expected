(program
 (define b (void))
 (set-global!
  b
  (lambda () (call/ec (lambda (return) (let ((a (void))) (set! a 3)))))))

